addons/udes_stock/models/stock_picking.py

```diff
@@ -994,3 +994,17 @@ def button_validate(self):
         res = super().button_validate()
         user_validating.unassign_pickings_from_users()
         return res
+
+    def unlink(self):
+        """
+        Override unlink to ensure that if disable_state_check context isn't set and there are
+        transfers in state different from draft to raise  error with appropriate message.
+        """
+        if not self._context.get("disable_state_check"):
```

Comment:
Suggestion (feel free to resolve): `bypass_state_check` as we aren't disabling anything, just bypassing it on certain occasions.

Reply:
I think you are right, replacing the name. Didnt pay attention on naming